CONTRIBUTING.md

```diff
@@ -9,23 +9,32 @@ There are 4 main ways of contributing to the PyMC project (in descending order o
 * Contributing or improving the documentation (`docs`) or examples (`pymc/examples`)
 * Submitting issues related to bugs or desired enhancements
 
-# Opening issues
+## Opening issues
 
 We appreciate being notified of problems with the existing PyMC code. We prefer that issues be filed the on [Github Issue Tracker](https://github.com/pymc-devs/pymc/issues), rather than on social media or by direct email to the developers.
 
 Please verify that your issue is not being currently addressed by other issues or pull requests by using the GitHub search tool to look for key words in the project issue tracker.
 
 Filter on the ["beginner friendly"](https://github.com/pymc-devs/pymc/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) label for issues which are good for new contributors.
 
-# Contributing code via pull requests
+## Etiquiette for code contributions
+When you start working working on an issue, open a `Draft` pull request as soon as you're doing the first steps.
+Make sure to mention the issue number in the PR description or commit message.
+This way others can see that the issue is being worked on.
+
+Comments like "I want to work on this issue" will be ignored.
```

Comment:
They won't exactly be ignored... I guess what you want to convey is that an issue is not exclusive to someone else, just because they commented on it first. I think it is still polite to ask the original person if they are still planning to work on it or if they can take over. I've seen several people saying "I don't have time, thanks for asking. Yes take over"

Reply:
Exactly as you say, recurring contributors don't have the need to do this, because they know they will open a PR fast enough to not be worth it. But I think asking for "permission" is still something new contributors feel the need for, and I don't think it's necessarily bad

Reply:
I commented https://github.com/pymc-devs/pymc/pull/5611#discussion_r829896757 trying to not change the meaning. We should first agree on that before working on the wording. 

In case it helps, here is ArviZ guidance: https://arviz-devs.github.io/arviz/contributing/contributing_prs.html